### PR TITLE
line 3 added by tech

### PR DESCRIPTION
**Pull request by psureshtech**

### CODE REVIEW

**Comment by psureshdev** [comment]
nice please add line 4

**Comment by psureshtech** [comment]
done

File: apple [hunk 1]
```diff
@@ -1,2 +1,3 @@
 line 1
 line 2
+line 3 by tech
```
Comment (psureshdev) [inline apple:3]: done add line 4

Comment (psureshtech) [reply apple:3]: done

File: apple [hunk 2]
```diff
@@ -1,2 +1,5 @@
...
+line 4 by tech
+f
```
Comment (psureshdev) [inline apple:5]: nice add 5

Comment (psureshtech) [reply apple:5]: okay

**Pull request merged**